Atomic/scan.py

```diff
@@ -170,11 +170,12 @@ def gen_images():
         def gen_containers():
             slist = []
             for con in self.get_containers():
-                if con['Status'].upper() != 'DEAD':
-                    con['input'] = con['Id']
-                else:
-                    raise ValueError("The container ID {} is Dead and therefor cannot be scanned.".format(con['Id'][:12]))
-                slist.append(con)
+                if 'Status' in con:
```

Comment:
Thing is though when I try using the .get when there is a none type it says none type has no attribute called upper.